packages/styles/globals.import.styl

```diff
@@ -0,0 +1,148 @@
+html
+body
+  height 100%
+
+body
+  @extend .body-font
+  color $body-text
+
+h1
+h2
+h3
+h4
+  @extend .header-font
+  color $primary
+  font-weight 700
+
+h1
+  font-size 42px
+  font-weight 800
+
+h2
+  margin-bottom 20px
+  margin-top 0
+  font-size 32px
+  color $primary
+
+h2.section-title
+  margin-top 0
+  font-size 40px
+
+h2 a.more-info
+  font-size 12px
+  color $btn-default
+
+h3
+  margin-top 0
+  margin-bottom 15px
```

Comment:
I'm sure there was reasoning in GRRS. I'll review this so we start with a cleaner slate.